src/en/tut-lxd.md

```diff
@@ -0,0 +1,264 @@
+Title: Getting started with Juju and LXD
+
+# Getting started with Juju and LXD
+
+[LXD](lxd-upstream) provides an amazing experience using machine containers to be able to
+create the same models that are created with Juju on other public and private
+clouds. Using LXD as a method to test, verify, and replicate complex software
+deployments is a powerful tool that every Juju user needs. These instructions
+will get you up and running and deliver the best-possible experience with
+Juju. At the moment, that means using the latest release of Ubuntu with '[Long
```

Comment:
Sometimes the markdown parser can choke on links with linebreaks in them. It is pretty much always better to begin a new line for long links, so:

      '[Long Term Support][long-term-support]' (LTS): 
      [16.04 LTS (Xenial)][Xenial-download]